plugin/loader/preload_list

```diff
@@ -8,3 +8,4 @@ ipldgit github.com/ipfs/go-ipfs/plugin/plugins/git *
 badgerds github.com/ipfs/go-ipfs/plugin/plugins/badgerds *
 flatfs github.com/ipfs/go-ipfs/plugin/plugins/flatfs *
 levelds github.com/ipfs/go-ipfs/plugin/plugins/levelds *
+peerlog github.com/ipfs/go-ipfs/plugin/plugins/peerlog *
```

Comment:
This shouldn't be added to the default preload list. We need to add a way to preload plugins on build.